src/indexOf.js

```diff
@@ -21,6 +21,12 @@ var _isArray = require('./internal/_isArray');
  *
  *      R.indexOf(3, [1,2,3,4]); //=> 2
  *      R.indexOf(10, [1,2,3,4]); //=> -1
+ *
+ *      var f = function() {};
+ *      var g = function() {};
+ *      var h = fucntion() {};
```

Comment:
`s/fucntion/function/`

I wonder, though, whether we should include these examples on the website. They're very much worth including in the test suite, but the examples on the website should demonstrate common usage. Finding the index of a function in a list of functions is not a common operation, I would say. What do others think?